rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -27,13 +29,67 @@ local function fetch_request_body()
   return body
 end
 
+-- Returns the certificate and key for a given host
+function _M.get_cert_key(hostname)
+  return configuration_data:get(hostname)
+end
+
+local function handle_cert_request()
```

Comment:
since this is handling requests to `/configuration/servers`, let's call it `handle_servers`